plugins/lsp/lsp-status.nix

```diff
@@ -0,0 +1,83 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "lsp-status";
+  originalName = "lsp-status.nvim";
+  url = "https://github.com/nvim-lua/lsp-status.nvim";
+  defaultPackage = pkgs.vimPlugins.lsp-status-nvim;
+  maintainers = [ helpers.maintainers.b3nb5n ];
+  settingsOptions = with lib; {
+    plugins.lsp-status = rec {
+      enable = mkEnableOption "lsp-status.nvim";
```

Comment:
Settings options are exclusively for config passed to the `setup` lua function. You don't need `enable` and `package` here, `mkNeovimPlugin` will create those options automatically _outside_ of the `settings` option.